### ISSUE TITLE
Running optional to nullsafe introduces bug

### ISSUE THREAD

**Issue by rzv-me** [laravel-shift/workbench-issues#58]
**Desktop:**
 - OS: Mac
 - OS Version: Big Sur 11.6
 - App Version: 0.7.2

**Describe the bug**
If you run the shift `optional to nullsafe` on some code, it removes the optional but doesn't add the nullsafe operator where needed.

**To Reproduce**
1. Run 'optional to nullsafe' on the following code

```php
<?php

namespace App\Actions;

use Illuminate\Console\Command;
use Lorisleiva\Actions\Concerns\AsAction;

class CleanRedisSessions
{
    use AsAction;

    public function handle($limit = 10000, Command $command = null): void
    {
        $command = optional($command);
        collect(str_split('abcdefghijklmnopqrstuvwxyz'))->each(function ($char) use ($limit, $command) {
            $command->info('This is for ' . $char);
        });
    }

    public function asCommand(Command $command): void
    {
        $command->info('limit = '.$command->option('limit'));
        $this->handle($command->option('limit'), $command);
        $command->info('Done!');
    }
}
```

2. Notice that $command can be null, but the shift removes the optional from it. So the $command->info in the function can fail, if command is null

**Comment by jasonmccreary** [comment]
This is a probably an edge case where Shift should not convert `optional` since it leverages the underlying delay evaluation.

@jessarcher, going to assign this one to you as it should be a quick fix with the test case and parser. However, we might want to note it in a "comment" as a remaining instance of `optional` to be reviewed.

**Comment by jasonmccreary** [comment]
This should be resolved in the recent release (`0.7.5`). Note, the Workbench no longer converts these assignments as they leverage the "lazy execution" behavior of `optional`.
